### PR TITLE
[Coding Style] Enable rule PSR12.Properties.ConstantVisibility

### PR DESCRIPTION
**Pull request by michalkleiner**
### Description:

No file changes necessary when enabling this rule.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by sgiehl — CHANGES_REQUESTED** [review]
There are changes required, the rule just don't support autofix with phpcfb

**Comment by michalkleiner** [comment]
> There are changes required, the rule just don't support autofix with phpcfb

Of course, you're right. Got caught up by this before and forgot about it. Will address that.

**Comment by michalkleiner** [comment]
Made all the constants public (as they were technically), at some point we can review which could/should be private.

File: libs/HTML/QuickForm2/Controller.php [hunk 1]
```diff
@@ -70,12 +70,12 @@ class HTML_QuickForm2_Controller implements IteratorAggregate
    /**
     * Key in $_REQUEST array that contains the ID of the Controller
     */
-    const KEY_ID = '_qfc_id';
+    public const KEY_ID = '_qfc_id';
```
Comment (sgiehl) [inline libs/HTML/QuickForm2/Controller.php]: We shouldn't apply changes in `/libs`. The folder is excluded from phpcs check anyway

Comment (michalkleiner) [reply libs/HTML/QuickForm2/Controller.php]: Reverted all changes in the `/libs` folder.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**